### ISSUE TITLE
Convert YOLACT to OpenVINO

### ISSUE THREAD

**Issue by felipesanmartin** [openvinotoolkit/open_model_zoo#1917]
Hi everyone, I have been trying to convert the YOLACT model to OpenVINO using the tools (downloader.py and converter.py) and the PR pushed by @eaidova, but I have many errors.
I have changed the versions of PyTorch and OpenVINO, the last configuration I tried was OpenVINO 2020.3.1 and PyTorch 1.5.0 getting the following error:

`[ ERROR ]  Shape is not defined for output 0 of "544".
[ ERROR ]  Cannot infer shapes or values for node "544".
[ ERROR ]  Not all output shapes were inferred or fully defined for node "544". 
 For more information please refer to Model Optimizer FAQ (https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_Model_Optimizer_FAQ.html), question #40. 
[ ERROR ]  
[ ERROR ]  It can happen due to bug in custom shape infer function <function Slice.infer at 0x7f050f608598>.
[ ERROR ]  Or because the node inputs have incorrect values/shapes.
[ ERROR ]  Or because input shapes are incorrect (embedded to the model or passed via --input_shape).
[ ERROR ]  Run Model Optimizer with --log_level=DEBUG for more information.
[ ERROR ]  Exception occurred during running replacer "REPLACEMENT_ID" (<class 'extensions.middle.PartialInfer.PartialInfer'>): Stopped shape/value propagation at "544" node. 
 For more information please refer to Model Optimizer FAQ (https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_Model_Optimizer_FAQ.html), question #38. 
FAILED:
yolact-resnet50-fpn-pytorch
`
Can you tell me how to convert the model? Maybe I'm using a bad combination of libraries or something like that.

I'll be grateful for your help,
Felipe San Martín

**Comment by eaidova** [comment]
Hi @felipesanmartin, we tested model conversion with openvino 2021.1 and higher, pytorch 1.5 is fine. it is possible that that your openvino version does not fully support all necessary layers for performing model conversion.
Could you please try openvino 2021.1?

**Comment by felipesanmartin** [comment]
I could converter the model using OpenVINO 2021.1, thanks a lot @eaidova!

**Comment by jsn5** [comment]
@felipesanmartin Hi, did you try converting the default model trained on coco dataset with 80 classes or custom data with different number of classes? I'm trying to convert a yolact_restnet50 model trained on custom data with 1 class, but fails at the onnx conversion part.

**Comment by felipesanmartin** [comment]
Hi @jsn5, sorry for the lateness of my answer. I converted a modified version of yolact, using 1 class and mobilenet-v2 as backbone. Basically, I downloaded the model using the utility from this repo  and modified some files, such as 'data/config.py', 'backbone.py' and 'yolact.py'. Also, I had to modify a yml file and 'model.py'. I don't remember the specific procedure, but it's not so hard. I'm going to try to write a post with all the process and share here.
